integr8ly/cloud-resource-operator
# generate cluster service versions

## Overview
Generate csv updates, pulling in changes in the operator.yaml, the roles and cluster roles and crds dynamically, avoiding manual error. 

## Verification
```
Note: operator-sdk version must be 0.12.0 for this to work as expected
```

- Clone this branch
- Push a new cro manifest from this branch to your quay.io app registry
- Ensure that the cloud resource operator is deployed as expected through OLM
- Ensure that any cro resources can be provisioned as expected
